redi-connect-front/src/components/organisms/PreFooter.tsx

```diff
@@ -7,38 +7,50 @@ import {
   Content,
   Image
 } from 'react-bulma-components'
+import { withRouter } from 'react-router-dom'
 import Button from '../atoms/Button'
 import hello from '../../assets/images/hello.svg'
-// import helloMobile from "../../assets/images/hello-mobile.svg";
+import helloMobile from '../../assets/images/hello-mobile.svg'
```

Comment:
Are the svgs for mobile and desktop the same? 
The solution would mean that you are loading both images always. 

There is also an old-school solution possible with floats, but I'm not that strong on it. It would be good to find a way not to load both image always.

Reply:
They are different ... the hello text bubble is aligned to the middle and to the right